package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "cdh-web",
+  "description": "Django web application for the CDH Website",
+  "repository": "https://github.com/Princeton-CDH/cdh-web.git",
+  "author": "The Center for Digital Humanities at Princeton <cdhdevteam@princeton.edu>",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "autoprefixer": "^9.1.3",
+    "postcss-cli": "^6.0.0"
+  },
+  "scripts": {
+    "pa11y": "pa11y-ci --sitemap http://localhost:8000/sitemap.xml"
+  },
```

Comment:
I can use this locally by changing my site domain name to `localhost:8000` in the admin...is there any downside to doing that?

Reply:
I don't understand the question/concern here

Reply:
I just don't know what other things changing the site domain would affect since I don't know much about the sites framework / what things reference it other than the sitemaps. If it does nothing else, great!